cylc/flow/config.py

```diff
@@ -690,7 +690,7 @@ def process_initial_cycle_point(self) -> None:
         Sets:
             self.initial_point
             self.cfg['scheduling']['initial cycle point']
-            self.options.icp
+            self.raw_icp
```

Comment:
I don't find "raw" to be particularly illuminating. Can we avoid using a separate attribute and just use `self.initial_point` for this?